taskcluster/focusios_taskgraph/transforms/bitrise.py

```diff
@@ -0,0 +1,102 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+"""
+Resolve secrets and dummy secrets
+"""
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+from taskgraph.transforms.base import TransformSequence
+
+
+transforms = TransformSequence()
+
+
+@transforms.add
+def set_run_config(config, tasks):
+    for task in tasks:
+        run = task.setdefault("run", {})
+        run.setdefault("using", "run-commands")
+        run.setdefault("use-caches", False)
+
+        run["secrets"] = {
+            "by-level": {
+                "3": [{
+                    "name": "project/mobile/focus-ios/bitrise",
+                    "key": "api_key",
```

Comment:
Not sure whether we have to add this key also for focus-iOS

Reply:
@escapewindow would this be fine or do we need that key?

Reply:
@isabelrios we do have to create this key.
Do we create a new key or reuse firefox-ios' key? If we create new, who can do that?

Reply:
I think we need to create a new one for focus. I can do that by creating a token on bitrise and then we should put it in the secret. But sorry for my ignorance there, don't know where that secret lives... Do you know how to do that? I can create the token and share it with you via Keybase

Reply:
For your reference, this is how it was set for firefox-iOS: https://github.com/mozilla-mobile/firefox-ios/pull/6337#discussion_r401685892

Reply:
Yes, I'm able to create/update the secret. It'll live in https://firefox-ci-tc.services.mozilla.com/secrets , like https://firefox-ci-tc.services.mozilla.com/secrets/project%2Fmobile%2Ffirefox-ios%2Fbitrise . If we want other people to be able to create or update those secrets, we can grant them scopes.

Reply:
Great, I will share the token with you, let me know other tool if you prefer. 
We have only needed this once for each product, I would say no need for now to grant us or other people in the team scopes and as long as you don't mind doing it. Better to have that kind or restricted I guess

Reply:
@escapewindow I have just sent it you

Reply:
The secret is created, thank you!

Reply:
cool, thanks! I think that would be all.. I will land this PR soon and give it a try 🤞